README.md

```diff
@@ -0,0 +1,70 @@
+# GPS Visualiser
+
+Summarisation of the findings in [Global Pneumococcal Sequencing Project](https://www.pneumogen.net/gps/) (GPS) country-specific analysis papers in a visual format.
+
+This summarisation effort is composed of a back-end Python script and a front-end website.
+
+&nbsp;
+## Back-end Python Script
+The Python script `processor.py` extracts and processes data from the GPS database (not publicised) and export a JSON file `data.json` to be taken up by the front-end website.
```

Comment:
Does the data.json contain non-published data?

Reply:
@blue-moon22  No. All data that is not marked as 'Y' in the 'Published(Y/N)' column is dropped. 

```
    # Only preserving rows with published data
    df.drop(df[df['Published(Y/N)'] != 'Y'].index, inplace=True)
```